### PR TITLE
Initial actions/labeler config

### PR DESCRIPTION
**Pull request by zvecr**
## Description

A stab at adding some labels automagically, as some first steps at some process automation.

Automatic keyboard label requires <https://github.com/actions/labeler/issues/9> otherwise something like `keyboards/**/*` would also be triggered by keymap changes.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Automation for labelling, heck yeah

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+core:
```
Comment (drashna) [inline .github/labeler.yml]: Should probably add `util` to this, for the time being.

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - drivers/**/*
+  - tests/**/*
+keymap:
+  - users/**/*
+  - layouts/**/*
+  - keyboards/**/keymap/**/*
+cli:
+  - bin/qmk
+  - requirements.txt
+  - lib/python/**/*
```
Comment (drashna) [inline .github/labeler.yml]:
```suggestion
  - lib/python/**/*
python:
  - lib/python/**/*
  - *.py
```

This too?

File: .github/labeler.yml (hunk 1 above)
Comment (zvecr) [reply .github/labeler.yml]: yeah, stuff like build scripts live in the folder so makes sense.

File: .github/labeler.yml (hunk 2 above)
Comment (zvecr) [reply .github/labeler.yml]: Done!

**Pull request merged**